tasks/ping_pong_task/server_ping_pong_task.py

```diff
@@ -232,7 +233,7 @@ def _to_client_update_state(self):
             data["seconds"] = 1 if seconds_to_send <= 0 else seconds_to_send
 
             # Record state of the game
-            self._csv_writer.writerow([time(), json.dumps(data)])
+            self._csv_writer.writerow([time.time(), time.monotonic(),  psutil.boot_time(), json.dumps(data)])
```

Comment:
We should label each column in the CSV file in the init function of this class. Simply write the first row as ["time", "monotonic_time", "boot_time", "image_path", "state"]